apps/showcase/src/app/configuration/configuration.component.ts

```diff
@@ -0,0 +1,72 @@
+import { CommonModule } from '@angular/common';
+import { AfterViewInit, ChangeDetectionStrategy, Component, QueryList, ViewChildren, ViewEncapsulation } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { ApplicationDevtoolsModule } from '@o3r/application';
+import { ComponentsDevtoolsModule } from '@o3r/components';
+import { ConfigurationBaseServiceModule, ConfigurationDevtoolsMessageService, ConfigurationDevtoolsModule } from '@o3r/configuration';
+import { O3rComponent } from '@o3r/core';
+import { ConfigurationPresComponent, CopyTextPresComponent, IN_PAGE_NAV_PRES_DIRECTIVES, InPageNavLink, InPageNavLinkDirective, InPageNavPresService } from '../../components/index';
+import { ConfigurationPresConfig } from '../../components/showcase/configuration/configuration-pres.config';
+
+const CONFIG_OVERRIDE: ConfigurationPresConfig = {
+  inXDays: 30,
+  destinations: [
+    { cityName: 'Manchester', available: true },
+    { cityName: 'Nice', available: true },
+    { cityName: 'Dallas', available: true }
+  ],
+  shouldProposeRoundTrip: true
+};
+
+@O3rComponent({ componentType: 'Page' })
+@Component({
+  selector: 'o3r-configuration',
+  standalone: true,
+  imports: [
+    CommonModule,
+    RouterModule,
+    ConfigurationPresComponent,
+    ApplicationDevtoolsModule,
+    ComponentsDevtoolsModule,
+    ConfigurationDevtoolsModule,
+    ConfigurationBaseServiceModule,
+    CopyTextPresComponent,
+    IN_PAGE_NAV_PRES_DIRECTIVES
+  ],
+  templateUrl: './configuration.template.html',
+  styleUrls: ['./configuration.style.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ConfigurationComponent implements AfterViewInit {
+  @ViewChildren(InPageNavLinkDirective)
+  private inPageNavLinkDirectives!: QueryList<InPageNavLink>;
+  public links$ = this.inPageNavPresService.links$;
+
+  public config: ConfigurationPresConfig | undefined = undefined;
+
+  public get codeConfig() {
+    return this.config ? `<o3r-configuration-pres
+  [config]="${JSON.stringify(this.config, null, 2)}"
+></o3r-configuration-pres>` : '<o3r-configuration-pres></o3r-configuration-pres>';
+  }
```

Comment:
may that would be a good place to use `signals` 
I am not a big fan of the custom getter being called inside the template